spec/models/video_spec.rb

```diff
@@ -7,6 +7,7 @@
   it { should have_many(:reviews).order("created_at DESC") }
 
   describe "search_by_title" do
```

Comment:
This describe block's message should be slightly different - Read more [here](http://www.betterspecs.org/#describe).